docs/specs/discovery.md

```diff
@@ -34,18 +36,21 @@ sequenceDiagram
   %%{wrap}%%
   participant Devices
   participant Spotter
-  participant Agent as Agent<br/>(w/ Mapping)
+  participant Registry
+  participant Provisioning Engine
+  participant Mapping Agent
   participant Pipeline
-  Note over Devices, Agent: Discovery Start
-  activate Agent
-  Agent->>Spotter: DISCOVERY CONFIG<br/>()
+  Note over Devices, Provisioning Engine: Discovery Start
+  activate Provisioning Engine
+  Mapping Agent->>Registry: DISCOVERY CONFIG<br/>for Spotter<br/>()
+  Registry->>Spotter: DISCOVERY CONFIG
   loop
     Devices-->Spotter: fieldbus
-    Spotter->>Agent: DISCOVERY EVENT<br/>(*scan_id)<br/><properties: *uniqs>
+    Spotter->>Provisioning Engine: DISCOVERY EVENT<br/>(*scan_id)<br/><properties: *uniqs>
```

Comment:
I don't think this is `uniqs` anymore -- what's actually in the payload now... I think it might be `refs`... ?